frontend/packages/knative-plugin/src/utils/create-knative-utils.ts

```diff
@@ -6,6 +6,7 @@ import {
   ConfigurationModel,
   RouteModel,
 } from '@console/knative-plugin';
+import { getAppLabels } from '@console/dev-console/src/utils/resource-label-utils';
```

Comment:
Just FYI.
For now we can live with this dependency. However we should improve the relationship between knative-plugin and dev-console. IMO dev-console should expose API and have no direct dependency on knative-plugin. In which case this dependency is ok. But right now we have cross package dependencies due to dev-console not exposing extensions to contribute to the import flows.